src/components/PostcodeLookup/PostcodeLookup.js

```diff
@@ -228,13 +228,15 @@ class PostcodeLookup extends Component {
   createAddressDropdownList() {
     const addresses = [{ label: 'Please select', value: null }];
     this.state.addressLookupData.map(item =>
-      addresses.push({ label: item.Line1, value: item }));
+      addresses.push({ label: typeof item.Line2 === 'undefined' ? item.Line1 : `${item.Line1}, ${item.Line2}`,
```

Comment:
I take it this label is being used as a key at a later point?

Reply:
To make it truly unique, and if the key is not being use as anything other than a reference, you could use shortID? https://www.npmjs.com/package/shortid